src/widgets/medications/medication-order.component.tsx

```diff
@@ -42,10 +42,10 @@ export default function MedicationOrder(props: MedicationOrderProps) {
   const [dosingInstructions, setDosingInstructions] = useState("");
   const [drugStrength, setDrugStrength] = useState<number>(null);
   const [startDate, setStartDate] = React.useState(
-    dayjs(new Date()).format("DD-MMM-YYYY")
+    dayjs(new Date()).format("YYYY-MM-DD")
```

Comment:
Here we fix the format, but shouldn't we actually take the format from the user's locale / region?

Reply:
How do you recommend we use the locale format?